tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/division_signpost_block.html

```diff
@@ -1,20 +1,27 @@
 {% load wagtailcore_tags wagtailimages_tags %}
-<div>
-    {# Section heading #}
-    <h2>{{ value.title }}</h2>
-
-    {# Section intro #}
-    <div>{{ value.intro|richtext }}</div>
-
-    {# Signposting cards #}
-    <ul>
-        {% for card in value.cards %}
-            <li class="{{ card.card_colour }}">
-                {% firstof card.heading card.page.title %}
-                {{ card.description|richtext }}
-                {% image card.image fill-100x100 %}
-                <a href="{% pageurl card.page %}">{{ card.link_text }}</a>
-            </li>
-        {% endfor %}
-    </ul>
+<div class="grid division-signpost">
+    <div class="grid__division-signpost">
+        {# Section heading #}
+        {% if value.title %}
+            <h2 class="heading heading--two">{{ value.title }}</h2>
+        {% endif %}
+        {# Section intro #}
+        {% if value.intro %}
+            <p class="text text--five division-signpost__intro">{{ value.intro|richtext }}</p>
```

Comment:
Suggestion: If this is `|richtext` then it needs to be output in a `div` rather than a `p` tag - i'm pretty sure that otherwise the `<p>` tag is empty and the rich text markup is output in it's own tags underneath.

Reply:
good spot! done